frontend-react/src/pages/submissions/SubmissionsTable.tsx

```diff
@@ -0,0 +1,57 @@
+import { useContext } from "react";
+import moment from "moment";
+import { useResource } from "rest-hooks";
+
+import SubmissionsResource from "../../resources/SubmissionsResource";
+import { GlobalContext } from "../../components/GlobalContextProvider";
+
+function SubmissionsTable() {
+    // this component will refresh when global context changes (e.g. organization changes)
+    const globalState = useContext(GlobalContext);
+    const submissions: SubmissionsResource[] = useResource(
+        SubmissionsResource.list(),
+        { organization: globalState.state.organization }
+    );
+    return (
+        <div className="grid-container usa-section margin-bottom-10">
```

Comment:
Would be nice to migrate over to the trussworks <Grid> classes at some point. But not required here.